<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormHeadLeft</class>
 <widget class="QWidget" name="FormHeadLeft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>94</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>568</width>
    <height>94</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>568</width>
    <height>94</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>32</y>
     <width>234</width>
     <height>26</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>234</width>
     <height>26</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>26</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:24px;
color:#1d2e38;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>0000-00-00 00:00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitle">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>29</y>
     <width>326</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>326</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:48px;
color:#373f44;</string>
   </property>
   <property name="text">
    <string>启动中</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>-1</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCheckDetail">
   <property name="geometry">
    <rect>
     <x>486</x>
     <y>51</y>
     <width>70</width>
     <height>35</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>35</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font-size:18px;
	color:#373f44;
	border-radius:5px;
	background-image:url(:/public/header/check_back.png);
}
QPushButton:pressed{
	background-image:url(:/public/header/check_back_clicked.png);
}</string>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </widget>
  <zorder>labelTitle</zorder>
  <zorder>labelTime</zorder>
  <zorder>pushButtonCheckDetail</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
